atomic_reactor/plugins/pre_add_labels_in_df.py

```diff
@@ -81,7 +81,8 @@ def __init__(self, tasker, workflow, labels,
             raise RuntimeError("labels have to be dict")
         self.labels = labels
         self.dont_overwrite = dont_overwrite
-        self.aliases = aliases or {}
+        default_aliases = {k: v for k, v in _PREFERRED_LABELS}
```

Comment:
Okay, not sure if I should update `_PREFERRED_LABELS` or there is a better place for it